README.md

```diff
@@ -16,3 +16,13 @@ Documentation
 =============
 
 http://jwcrypto.readthedocs.org
+
+Deprecation Notices
+===================
+
+2020.12.11: The RSA1_5 algorithm is now considered deprecated due to
+numerous implementation issues that make it a very problematic tool to
+use safely.
+The algorithm can still be used but requires explicitly allowing it on
+object instantiation. If your application depends on it there are examples
+of how to re-enable RSA1_5 usage in the tests files.
```

Comment:
I think we can state explicitly here that re-enabling it will make your application vulnerable if an attacker can force it to process arbitrary ciphertexts.